### PR TITLE
Update all dependencies

### PR DESCRIPTION
**Pull request by baurmatt**
## WHAT

Updated all dependencies

## WHY

Was building on may Apple M1.

```bash
$ go build
# golang.org/x/sys/unix
../../go/pkg/mod/golang.org/x/sys@v0.0.0-20200121082415-34d275377bf9/unix/syscall_darwin.1_13.go:25:3: //go:linkname must refer to declared function or variable
../../go/pkg/mod/golang.org/x/sys@v0.0.0-20200121082415-34d275377bf9/unix/zsyscall_darwin_arm64.1_13.go:27:3: //go:linkname must refer to declared function or variable
../../go/pkg/mod/golang.org/x/sys@v0.0.0-20200121082415-34d275377bf9/unix/zsyscall_darwin_arm64.1_13.go:40:3: //go:linkname must refer to declared function or variable
../../go/pkg/mod/golang.org/x/sys@v0.0.0-20200121082415-34d275377bf9/unix/zsyscall_darwin_arm64.go:28:3: //go:linkname must refer to declared function or variable
../../go/pkg/mod/golang.org/x/sys@v0.0.0-20200121082415-34d275377bf9/unix/zsyscall_darwin_arm64.go:43:3: //go:linkname must refer to declared function or variable
../../go/pkg/mod/golang.org/x/sys@v0.0.0-20200121082415-34d275377bf9/unix/zsyscall_darwin_arm64.go:59:3: //go:linkname must refer to declared function or variable
../../go/pkg/mod/golang.org/x/sys@v0.0.0-20200121082415-34d275377bf9/unix/zsyscall_darwin_arm64.go:75:3: //go:linkname must refer to declared function or variable
../../go/pkg/mod/golang.org/x/sys@v0.0.0-20200121082415-34d275377bf9/unix/zsyscall_darwin_arm64.go:90:3: //go:linkname must refer to declared function or variable
../../go/pkg/mod/golang.org/x/sys@v0.0.0-20200121082415-34d275377bf9/unix/zsyscall_darwin_arm64.go:105:3: //go:linkname must refer to declared function or variable
../../go/pkg/mod/golang.org/x/sys@v0.0.0-20200121082415-34d275377bf9/unix/zsyscall_darwin_arm64.go:121:3: //go:linkname must refer to declared function or variable
../../go/pkg/mod/golang.org/x/sys@v0.0.0-20200121082415-34d275377bf9/unix/zsyscall_darwin_arm64.go:121:3: too many errors
```

### CODE REVIEW

**Review by drlau — CHANGES_REQUESTED** [review]
Thanks for your contribution! Can you also update the mock GitLab API in `notifier/gitlab/gitlab_test.go`? `s/OptionFunc/RequestOptionFunc`

**Comment by baurmatt** [comment]
Ah sorry, missed that! Done :)

File: notifier/gitlab/client.go [hunk 1]
```diff
@@ -2,7 +2,6 @@ package gitlab
 
 import (
 	"errors"
-	"net/http"
 	"os"
```
Comment (drlau) [inline notifier/gitlab/client.go:5]:
```suggestion
	"fmt"
	"os"
```

File: notifier/gitlab/client.go [hunk 2]
```diff
@@ -2,7 +2,7 @@ package gitlab
 
 import (
 	"errors"
-	"net/http"
+	"fmt"
```
Comment (drlau) [inline notifier/gitlab/client.go:5]:
```suggestion
```

**Review by drlau — APPROVED** [review]

**Pull request merged**